airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/parquet_parser.py

```diff
@@ -37,14 +41,19 @@ def parse_records(
         stream_reader: AbstractFileBasedStreamReader,
         logger: logging.Logger,
     ) -> Iterable[Dict[str, Any]]:
+        parquet_format = config.format[config.file_type] if config.format else ParquetFormat()
+        if not isinstance(parquet_format, ParquetFormat):
+            raise ValueError(f"Expected ParquetFormat, got {parquet_format}")  # FIXME test this branch!
```

Comment:
this still needs a test right? I don't see any tests where we instantiate a new ParquetParser.parse_records(), or a new parquet test scenario using the wrong format

Reply:
added the tests https://github.com/airbytehq/airbyte/pull/28320/files#diff-84c5ac857864155f0ad3990df048f428d7d57ec433b70e6e339eceb6c3027582R170